### PR TITLE
Fix state URL test

### PR DESCRIPTION
**Pull request by huwshimi**
Fixed state URL test to work with the React flag.

### CODE REVIEW

**Comment by huwshimi** [comment]
No QA.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3010/
Test PASSed.

**Comment by anthonydillon** [comment]
Nice small change, tests pass :+1:

File: jujugui/static/gui/src/test/test_ui_state.js [hunk 1]
```diff
@@ -1033,7 +1033,7 @@ describe('UI State object', function() {
         sectionB: {}
       }
     }, {
-      '/inspector/service123/unit/13/': {
+      '/inspector/service123/13/': {
```
Comment (hatched) [inline jujugui/static/gui/src/test/test_ui_state.js]: To bring this test up to the new functionality the metadata should contain `activeComponent: 'unit'` and then I think the original url is correct.

**Comment by huwshimi** [comment]
I had to update the way we build the URL as we ended up with a `//` after the component name in the URL. It's possible the browser was hiding this bug from us.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3025/
Test PASSed.

**Comment by hatched** [comment]
:+1: QA OK!
Thanks for fixing it so we don't have to go back and do it when de-flagging :)

**Comment by huwshimi** [comment]
Thanks @hatched and @anthonydillon :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**